content/id/docs/concepts/cluster-administration/monitoring.md

```diff
@@ -0,0 +1,167 @@
+membaca metrik-metrik itu memerlukan otorisasi melalui sebuah User, Group, atau 
+ServiceAccount dengan ClusterRole yang memperbolehkan mengakses `/metrics`.
+
+Sebagai contoh:
+
+```
+apiVersion: rbac.authorization.k8s.io/v1	
+kind: ClusterRole	
+metadata:	
+  name: prometheus	
+rules:	
+  - nonResourceURLs:	
+      - "/metrics"	
+    verbs:	
+      - get
+```
+
+## Siklus Hidup (_Lifecycle_) Metrik
+
+Alpha metric →  Stable metric →  Deprecated metric →  Hidden metric → Deletion
+
+Metrik-metrik _Alpha_ tidak memiliki jaminan stabilitas; dengan demikian mereka 
+dapat dimodifikasi atau dihapus kapan saja.
+
+Metrik-metrik _Stable_ dijamin tidak berubah (dijamin stabilitasnya); Secara khusus, stabilitas berarti:
+
+* metrik itu sendiri tidak akan dihapus (atau diganti namanya)
+* jenis metrik tidak akan dimodifikasi
+
+Metrik _Deprecated_ memberi penanda bahwa metrik tersebut suatu saat akan dihapus; untuk 
+menemukan versi mana, kamu perlu memeriksa anotasi, yang mencakup dari versi 
+Kubernetes mana yang metrik tersebut akan dianggap _deprecated_.
+
+Sebelum _deprecation_:
+
+```
+# HELP some_counter this counts things
+# TYPE some_counter counter
+some_counter 0
+```
+
+Sesudah _deprecation_:
+
+```
+# HELP some_counter (Deprecated since 1.15.0) this counts things
+# TYPE some_counter counter
+some_counter 0
+```
+
+Setelah metrik disembunyikan maka secara default metrik tidak dipublikasikan 
+untuk _scraping_ (pengambilan metrik). Untuk menggunakan metrik yang tersembunyi, kamu perlu mengganti (_override_)
+konfigurasi untuk komponen klaster yang relevan.
+
+Setelah metrik dihapus, metrik tidak dipublikasikan. Kamu tidak dapat mengubah 
+ini menggunakan _override_.
+
+## Melihat Metrik tersembunyi
+
+Seperti dijelaskan di atas, para admin dapat mengaktifkan metrik tersembunyi 
+melalui opsi pada baris perintah (_command line_) untuk _binary_ (program) tertentu. Ini ditujukan untuk 
+digunakan sebagai jalan keluar bagi para admin apabila mereka gagal memigrasi 
+metrik yang sudah _deprecated_ dalam rilis terakhir.
+
+Opsi `show-hidden-metrics-for-version` menunjukkan versi yang ingin kamu tampilkan 
+metrik _deprecated_-nya dalam rilis tersebut. Versi ini ditampilkan dalam bentuk x.y, 
+di mana x adalah versi _major_, y ​​adalah versi minor. Versi _patch_ tidak 
+diperlukan meskipun metrik dapat di_-deprecate_ dalam rilis _patch_, hal ini
+adalah karena kebijakan _deprecation_ untuk metrik berlaku terhadap rilis minor.
+
+Opsi tersebut hanya dapat menggunakan versi minor sebelumnya sebagai parameternya. Semua 
+metrik yang disembunyikan di versi sebelumnya akan dikeluarkan jika para admin 
+mengatur versi sebelumnya ke `show-hidden-metrics-for-version`. Versi yang 
+terlalu lama tidak diizinkan karena hal ini melanggar kebijakan metrik yang
+_deprecated_.
+
+Ambil metrik `A` sebagai contoh, di sini diasumsikan bahwa `A` sudah _deprecated_ 
+pada rilis 1.n. Menurut kebijakan metrik _deprecated_, kita dapat mencapai kesimpulan 
```

Comment:
```suggestion
pada rilis 1.n. Menurut kebijakan _deprecation_ metrik, kita dapat mencapai kesimpulan 
```
Let's wait for the above solution too.

Reply:
the preference makes sense, let' use them
```
metrics deprecation policy -> kebijakan _depecration_ metrik
metrics deprecated policy -> kebijakan untuk metrik yang sudah _deprecated_
```

Reply:
Ok!

Reply:
Ok!